src/components/Sidebar/Sidebar.tsx

```diff
@@ -10,7 +11,9 @@ class Sidebar extends React.Component {
         <ul className="remove-style margin-reset padding-h-normal text-c">
           <li className="padding-v-normal">
```

Comment:
we probably want the whole list item (i.e. including the icon) to be a link

Reply:
good catch, included icon